content/ja/blog/_index.md

```diff
@@ -2,5 +2,17 @@
 title: ブログ
 menu: { main: { weight: 50 } }
 outputs: [HTML, RSS]
-default_lang_commit: c0a5eea5d720b0e075efa87f99dcf58c89106268
+description: OpenTelemetry ブログ
+default_lang_commit: 94d77ab8cbfe5552a7cd68bf677be86c574a613a
 ---
+
+<script>
+    document.addEventListener("DOMContentLoaded", function () {
+        if (window.location.pathname.includes('/page/')) return;
+
+        var checkbox = document.getElementById("m-blog2025-check");
+        if (checkbox) checkbox.checked = true;
+        checkbox = document.getElementById("m-blog2024-check");
+        if (checkbox) checkbox.checked = true;
+    });
+</script>
```

Comment:
This doesn't work for the `ja` site, and probably won't work for other non-`en` sites.

Reply:
Will track via:

- #6523